packages/teams-js/src/internal/mediaUtil.ts

```diff
@@ -207,7 +207,7 @@ export function validateScanBarCodeInput(barCodeConfig: media.BarCodeConfig): bo
  *
  * @internal
  */
-export function validatePeoplePickerInput(peoplePickerInputs: people.PeoplePickerInputs): boolean {
+export function validatePeoplePickerInput(peoplePickerInputs?: people.PeoplePickerInputs): boolean {
```

Comment:
>? [](http://example.com/codeflow?start=60&length=1)

This function was already written to support an `undefined` parameter value, this change acknowledged this support (and, in fact, was already potentially called with `undefined` values)